src/airflow/airqo_etl_utils/airqo_utils.py

```diff
@@ -946,9 +946,9 @@ def process_airnow_data_for_api(data: pd.DataFrame) -> list:
         for _, data_row in data.iterrows():
             restructured_data.append(
                 {
-                    "tenant": "airqo",
+                    "tenant": str(Tenant.US_EMBASSY),
```

Comment:
tenant is always AIRQO. @NoahNsimbe 

It is the network name which changes.

In the near future, the network field will no longer be required. cc: @BenjaminSsempala

Reply:
Unless we still are just storing in the separate DBs (tenants) while also using the AIRQO DB (tenant) for storage of all? Some sort of redundancy? @NoahNsimbe

Reply:
thanks @Baalmart

Reply:
Welcome @NoahNsimbe 
Slight CORRECTION...
In the near future, the `tenant` field will NO LONGER be required. CC: @BenjaminSsempala